packages/big-design/src/components/Select/types.ts

```diff
@@ -38,3 +38,8 @@ export interface SelectOption<T> extends Omit<BaseItem, 'actionType'> {
 export interface SelectAction extends BaseItem {
   onActionClick(inputText: string | null): void;
 }
+
+export interface SelectOptionGroup<T> {
+  groupLabel: string;
```

Comment:
What does the `Dropdown` group name this?

Reply:
`Dropdown` names this `DropdownItemGroup`. 

Since `Dropdown` has the `item` attribute while `Select` has the `option` attribute, I took the naming convention from `Dropdown` and replaced `Item` with `Option`. Suggestions?

Reply:
I was more thinking about the naming for `groupLabel`.

Reply:
AH! Gotcha. Yeah, `Dropdown` calls this `label`, however since the `Select` component has a `label` attribute, I was attempting to avoid confusion. Is it your opinion that I should go ahead and call this `label`?

Reply:
We should probably keep these consistent so that we don't have to reteach consumers how to use a component.

Reply:
Agree 👍